webapp-django/crashstats/supersearch/tests/common.py

```diff
@@ -3,253 +3,6 @@
 from crashstats.crashstats.tests.test_models import Response
 
 
-SUPERSEARCH_FIELDS_MOCKED_RESULTS = {
```

Comment:
A while back, @peterbe and @adngdb did some code changes to move super search fields from being stored in Elasticsearch to being stored in a giant JSON file in the repository. We had to back that out because it was tied to upgrading Elasticsearch. Then either @peterbe or I re-implemented it as Python code.

Anyhow, long story short, we needed to mock it out before because it was stored in Elasticsearch and each environment had their own copy and we needed something for the tests. But when we moved it to Python-land, we should have nixed all that code because mocking it wasn't necessary anymore.

Anyhow, this doesn't include the `uuid` field I just exposed, so I either had to add that field to the mockery or remove the mockery. I chose the latter because that seemed like a better end result.